_episodes/04-pipefilter.md

```diff
@@ -43,8 +43,7 @@ octane.pdb    pentane.pdb   propane.pdb
 {: .output}
 
 Let's go into that directory with `cd` and run the command `wc *.pdb`.
-`wc` is the "word count" command:
-it counts the number of lines, words, and characters in files.
+`wc` is the "word count" command  that counts the number of lines, words, and characters in files.
```

Comment:
I don't see how this is an improvement.

Reply:
Sometimes when using "it" or "this" while writing, it takes the reader some time to figure out what "it" is so I thought that removing that word would make it more clear. But maybe that's just me.